### PR TITLE
feat: add the table and campaign icons

### PR DESCRIPTION
**Pull request by GusBaamonde**
#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

add-table-and-icons

### CODE REVIEW

File: assets/scss/modules/_images.scss [hunk 1]
```diff
@@ -22,6 +22,20 @@
   background-repeat: no-repeat;
 }
 
+///
+/// Mixin campaign status icons
+///
+
+@mixin icon-campaing($background-color) {
```
Comment (cbernat) [inline assets/scss/modules/_images.scss]: ❤️

**Comment by cbernat** [comment]
Did you try an example with very very long subject? For now I think perhaps showing ... would be enough. Or perhaps a line break. Check here https://projects.invisionapp.com/share/GPU9MUS5WY4#/screens/387473062 when campaign name is too long it breaks in more than one line.

**Comment by GusBaamonde** [comment]
> Did you try an example with very very long subject? For now I think perhaps showing ... would be enough. Or perhaps a line break. Check here https://projects.invisionapp.com/share/GPU9MUS5WY4#/screens/387473062 when campaign name is too long it breaks in more than one line.

text-long

**Comment by GusBaamonde** [comment]
ajuste-contenido

**Pull request merged**